<!DOCTYPE html>
<html class="writer-html5" lang="en" >
<head>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" /><meta name="author" content="jpmslima" /><link rel="canonical" href="https://jpmslima.github.io/Material/" />
      <link rel="shortcut icon" href="../img/favicon.ico" />
    <title>Files and References - Tutoriais BioinfoX</title>
    <link rel="stylesheet" href="../css/theme.css" />
    <link rel="stylesheet" href="../css/theme_extra.css" />
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.8.0/styles/github.min.css" />
    
      <script>
        // Current page data
        var mkdocs_page_name = "Files and References";
        var mkdocs_page_input_path = "Material.md";
        var mkdocs_page_url = "/Material/";
      </script>
    
    <!--[if lt IE 9]>
      <script src="../js/html5shiv.min.js"></script>
    <![endif]-->
      <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.8.0/highlight.min.js"></script>
        <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.8.0/languages/yaml.min.js"></script>
      <script>hljs.highlightAll();</script> 
</head>

<body class="wy-body-for-nav" role="document">

  <div class="wy-grid-for-nav">
    <nav data-toggle="wy-nav-shift" class="wy-nav-side stickynav">
    <div class="wy-side-scroll">
      <div class="wy-side-nav-search">
          <a href=".." class="icon icon-home"> Tutoriais BioinfoX
        </a><div role="search">
  <form id ="rtd-search-form" class="wy-form" action="../search.html" method="get">
      <input type="text" name="q" placeholder="Search docs" aria-label="Search docs" title="Type search term here" />
  </form>
</div>
      </div>

      <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="Navigation menu">
              <ul>
                <li class="toctree-l1"><a class="reference internal" href="..">Página Inicial</a>
                </li>
              </ul>
              <ul>
                <li class="toctree-l1"><a class="reference internal" href="../demo/">Representação Computacional de Biomoléculas</a>
                </li>
              </ul>
              <ul>
                <li class="toctree-l1"><a class="reference internal" href="../aboutus/">Sobre o grupo</a>
                </li>
              </ul>
              <p class="caption"><span class="caption-text">Tutoriais Iniciais de Bioinformática</span></p>
              <ul>
                  <li class="toctree-l1"><a class="reference internal" href="../basic1/">Inicial</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../101/">Introdução a linha de comando UNIX</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../102/">Manipulando dados na CLI</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../CLI-Colab/">Introdução a CLI e ao Google Colab</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../103/">Introdução ao GenBank</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../104/">Buscas de Similaridade</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../105/">Alinhamento entre pares de sequências</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../106-MSA/">Alinhamento Múltiplo de Sequências</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../107-NCBI-CLI/">NCBI/GenBank via CLI</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../108-BLAST-CLI/">BLAST via CLI</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../109-SQL/">Comandos Básicos em SQL</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../r-colab/">Introdução ao R usando o Google Colab</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../enzima-annot/">Integração de Bancos - Exemplo</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../KEGG/">Explorando Vias Metabólicas</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../110-BDs/">Bancos de Dados de Informação Proteica</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../111-Modelagem/">Modelagem Computacional de Proteínas</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../112-MEGAX/">Primeiros Passos no MEGA</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../113-Genomes/">Navegando por Genomas</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../video/">Videoaulas</a>
                  </li>
              </ul>
              <p class="caption"><span class="caption-text">Bioinformática Estrutural</span></p>
              <ul>
                  <li class="toctree-l1"><a class="reference internal" href="../basic2/">Inicial</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../slides/">Slides de Acompanhamento</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../tools/">Programas e Ferramentas</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../STR1-Visualizacao/">Visualização Computacional de Moléculas</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../STR2-pred/">Informações a partir da estrutura primária</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../STR22-Epitopos/">Predição de Epítopos</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../STR3-Model1/">Modelagem 101</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../STR4-Model2/">Modelagem 102</a>
                  </li>
                  <li class="toctree-l1"><a class="" href="../STR5-Model3.mkdocs gh-deploy --forcemd">Modeller-CLI</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../STR6-AlphaFold/">Métodos baseados em IA</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../STR7-MDA/">MDA</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../STR8-Aval/">Avaliação dos Modelos</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../STR9-EM/">EM/CryoEM fitting</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../STR10-RINs/">RINs</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../STR11-APBS/">APBS</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../STR12-Memb/">Proteínas de Membrana</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../STR13-Dock1/">Autodock (básico)</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../STR14-Dock2/">Docking</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../STR15-VS1/">Virtual Screening</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../STR23-MDIntro/">Introdução a Dinâmica Molecular</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../STR16-EGB1/">Modelos de Pequenas Moléculas</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../STR17-EGB2/">Análise conformacional</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../STR18-EGB3/">Docking Automático</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../STR19-EGB4/">Posicionamento Manual</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../STR20-EGB5/">Dinâmica Molecular</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../STR21-EGB6/">VS-2</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../comps/">Complements</a>
                  </li>
              </ul>
              <p class="caption"><span class="caption-text">Análise Evolucionária de Sequências (Evolução Molecular)</span></p>
              <ul>
                  <li class="toctree-l1"><a class="" href="../satsub-ptbr.md">Análise de Saturação de Substituições</a>
                  </li>
                  <li class="toctree-l1"><a class="" href="../bestfit-ptbr.md">Escolha do Modelo de Substituição</a>
                  </li>
              </ul>
              <p class="caption"><span class="caption-text">Diplomado - UChile</span></p>
              <ul class="current">
                  <li class="toctree-l1"><a class="reference internal" href="../Presentation/">Home</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../UC_slides/">Slides</a>
                  </li>
                  <li class="toctree-l1 current"><a class="reference internal current" href="#">Files and References</a>
    <ul class="current">
    <li class="toctree-l2"><a class="reference internal" href="#working-datasets">Working Datasets</a>
    </li>
    <li class="toctree-l2"><a class="reference internal" href="#programs-applications-and-tools">Programs, Applications and Tools</a>
    </li>
    <li class="toctree-l2"><a class="reference internal" href="#online-tools">Online Tools</a>
    </li>
    <li class="toctree-l2"><a class="reference internal" href="#additional-links">Additional Links</a>
    </li>
    <li class="toctree-l2"><a class="reference internal" href="#references">References</a>
    </li>
    </ul>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../First_Tutorial_ENG/">First Tutorial</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../Substitution_saturation_ENG/">Analysis of Substitution Saturation</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../ModelTest_ENG/">Test of the Best-Fit Models</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../MEGAX_ENG/">Using MEGAX</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../1st_BEAST/">Using BEAST</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../BEAST%2BEBSP_ENG/">Phylodynamics using EBSP</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../IQTree/">Using IQ-Tree</a>
                  </li>
              </ul>
      </div>
    </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">
      <nav class="wy-nav-top" role="navigation" aria-label="Mobile navigation menu">
          <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
          <a href="..">Tutoriais BioinfoX</a>
        
      </nav>
      <div class="wy-nav-content">
        <div class="rst-content"><div role="navigation" aria-label="breadcrumbs navigation">
  <ul class="wy-breadcrumbs">
    <li><a href=".." class="icon icon-home" aria-label="Docs"></a></li>
          <li class="breadcrumb-item">Diplomado - UChile</li>
      <li class="breadcrumb-item active">Files and References</li>
    <li class="wy-breadcrumbs-aside">
    </li>
  </ul>
  <hr/>
</div>
          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
            <div class="section" itemprop="articleBody">
              
                <p><img alt="UChile" src="../UChile.png" /></p>
<h1 id="files">Files</h1>
<h2 id="working-datasets">Working <em>Datasets</em></h2>
<p><strong>Tree Files for use in FigTree:</strong></p>
<ul>
<li><a href="https://drive.google.com/uc?export=download&amp;id=1CPz5isUp25TEY9ZHf9R4adBNu-IBn51c">Tree File</a></li>
<li><a href="https://drive.google.com/uc?export=download&amp;id=1FjK3rTahhtx7paGs0zemJh4OyNZgAk4B">Tree File 2</a></li>
</ul>
<p><strong>Datasets for multiple sequence alignment:</strong></p>
<ul>
<li><a href="https://drive.google.com/uc?export=download&amp;id=1nUkT_zqkvHoqR5x7N5ggLmnMNb_982GH">primatesNuc_1.fasta</a></li>
<li><a href="https://drive.google.com/uc?export=download&amp;id=1qJMyqjvd6SIz6SAeeggHZdPIb9do8CIg">primatesAA_1.fasta</a></li>
</ul>
<p><strong>For use in the analysis of substitution saturation:</strong></p>
<ul>
<li><a href="https://drive.google.com/uc?export=download&amp;id=1sPodmXGkPCxWu7t8CIZzHmumYS_0kX18">VertCOI.fas</a></li>
<li><a href="https://drive.google.com/uc?export=download&amp;id=1pDxXpGbrWawiwBnhf3sZxi4EFGA-4Di-">InvertebrateEF1a.fas</a></li>
<li><a href="https://drive.google.com/uc?export=download&amp;id=1d99-xXZDWTE6nhAuI439T5VzHhTbeL2P">SIV.fas</a></li>
</ul>
<p><strong>Selection tests</strong></p>
<ul>
<li><a href="https://drive.google.com/uc?export=download&amp;id=1GncrP4SzFtV79LogQ-btK18EIltQHRTL">Data I.zip</a></li>
</ul>
<blockquote>
<p><em>Additional tutorial.</em></p>
</blockquote>
<p><strong>For use in the BEAST+EBSP tutorial:</strong></p>
<ul>
<li><a href="https://drive.google.com/uc?export=download&amp;id=19xrCmUwQ94JglHMU5BMjoNEAXRuK_jl6">Tutorial_Zika_Filogenia.zip</a></li>
</ul>
<h2 id="programs-applications-and-tools">Programs, Applications and Tools</h2>
<ul>
<li><a href="http://www.megasoftware.net">MEGAX</a></li>
<li><a href="http://dambe.bio.uottawa.ca/DAMBE/dambe.aspx">DAMBE</a></li>
<li><a href="http://www.mbio.ncsu.edu/BioEdit/bioedit.html">Bioedit</a></li>
<li><a href="https://wiki.gnome.org/Apps/Gedit">gedit</a></li>
<li><a href="https://github.com/ddarriba/jmodeltest2">jModelTest</a></li>
<li><a href="https://github.com/ddarriba/prottest3">Prottest 3</a></li>
<li><a href="https://github.com/ddarriba/modeltest">ModelTest-ng</a></li>
<li><a href="https://www.beast2.org">BEAST2</a></li>
<li><a href="http://beast.bio.ed.ac.uk/downloads">BEAST 1.8</a></li>
<li><a href="http://clustal.org/download/current/">Clustal X</a></li>
<li><a href="http://abacus.gene.ucl.ac.uk/software/paml.html#download">PAMLX</a></li>
<li><a href="https://www.r-project.org">R</a></li>
<li><a href="http://www.ormbunkar.se/aliview/">Aliview</a></li>
<li><a href="http://tree.bio.ed.ac.uk/software/figtree/">FigTree</a></li>
<li><a href="http://www.jalview.org">Jalview</a></li>
<li><a href="http://trimal.cgenomics.org">TrimAL</a></li>
</ul>
<h2 id="online-tools"><em>Online</em> Tools</h2>
<ul>
<li><a href="http://bioinfo.icb.ufmg.br/taxontree/#x">TaxOnTree</a></li>
</ul>
<blockquote>
<p><em>Comparative and exploratory phylogeny, as well as phylogenetic trees annottion.</em></p>
</blockquote>
<ul>
<li><a href="http://tcoffee.crg.cat">T-Coffee</a></li>
</ul>
<blockquote>
<p><em>Alignment program. Several options depending on your dataset.</em></p>
</blockquote>
<ul>
<li><a href="http://phylemon2.bioinfo.cipf.es">Phylemon2</a></li>
</ul>
<blockquote>
<p><em>Site with on line phylogenetics tools and pipelines. There, you can use TrimAL to "clean" gappy alignments.</em></p>
</blockquote>
<ul>
<li><a href="https://www.datamonkey.org">DataMonkey</a></li>
</ul>
<blockquote>
<p><em>Several tools used in selection tests, recombination and gene conversion detection.</em></p>
</blockquote>
<ul>
<li><a href="http://pseudochecker.ciimar.up.pt/pseudochecker/index.html">Pseudochecker</a></li>
</ul>
<blockquote>
<p><em>A tool to infer eukaryotic nuclear protein-coding genes inactivation.</em></p>
</blockquote>
<ul>
<li><a href="http://www.phylo.org/">CIPRES</a></li>
</ul>
<blockquote>
<p><em>Public portal containing pipelines for inferring phylogenies.</em></p>
</blockquote>
<h2 id="additional-links">Additional <em>Links</em></h2>
<ul>
<li><a href="http://www.timetree.org">TimeTree</a></li>
<li><a href="http://www.phylogeny.fr/index.cgi">Phylogeny.fr</a></li>
<li><a href="https://nextstrain.org">Nextstrain</a></li>
<li><a href="https://www.gisaid.org">GSAID</a></li>
<li><a href="https://www.ncbi.nlm.nih.gov/genome/viruses/variation/">Virus Variation</a></li>
<li><a href="https://beast.community">BEAST 1.X Tutorials</a></li>
<li><a href="https://www.beast2.org/tutorials/">BEAST 2.X Tutorials</a></li>
</ul>
<h2 id="references">References</h2>
<ul>
<li>Anisimova M, Liberles DA, Philippe H, Provan J, Pupko T, Haeseler von A. State-of the art methodologies dictate new standards for phylogenetic analysis. BMC Evol Biol. 2013;13(1):1-1. doi:10.1186/1471-2148-13-161.</li>
<li>Higgs, P.G.; Attwood, T.K. Bioinformatics and Molecular Evolution.. Malden (Massachusetts): Blackwell Publishing. 2005.</li>
<li>Bouckaert, R. et al. BEAST 2: A Software Platform for Bayesian Evolutionary Analysis. PLoS Computational Biology, v. 10, n. 4, p. e1003537, 10 abr. 2014.</li>
<li>Choudhuri S. Bioinformatics for Beginners: Genes, Genomes, Molecular Evolution, Databases and Analytical Tools. Elsevier, 2014.</li>
<li>Cvrckova F. A plant biologists’ guide to phylogenetic analysis of biological macromolecule sequences. Biologia Plantarum. 2016;60(4):619-627.</li>
<li>Dickinson H, Costa L, Gutierrez-Marcos J. Epigenetic neofunctionalisation and regulatory gene evolution in grasses. Trends in Plant Science. 2012;17(7):389-394.</li>
<li>Do CB, Katoh K. Protein Multiple Sequence Alignment. Thompson JD, Schaeffer-Reiss C, Ueffing M, eds. 2008:379-413. doi:10.1007/978-1-59745-398-1_25.</li>
<li>Drummond, A. J. et al. Bayesian coalescent inference of past population dynamics from molecular sequences. Molecular Biology and Evolution, v. 22, p. 1185–1192, 2005. </li>
<li>Edgar RC, Batzoglou S. Multiple sequence alignment. Current Opinion in Structural Biology. 2006;16(3):368-373. doi:10.1016/j.sbi.2006.04.004.</li>
<li>Faria, N. R. et al. Zika virus in the Americas: Early epidemiological and genetic findings. Science (New York, N.Y.), v. 5036, n. March, p. aaf5036, 2016. </li>
<li>Futuyma DJ. Evolution. Sunderland, Massachusetts, USA.: Sinauer Associates; 2nd Edition. 2009.</li>
<li>Gardner PP. A benchmark of multiple sequence alignment programs upon structural RNAs. Nucleic Acids Research. 2005;33(8):2433-2439. doi:10.1093/nar/gki541.</li>
<li>Gibas, c. &amp; jambeck, P. Developing Bioinformatics Computer Skills. O ́Reilly &amp; Associates, Inc. USA. 2001. </li>
<li>Graur, d. &amp; Li, W. H. Fundamentals of Molecular Evolution. 2a edição. Sinauer Press, Sunderland, Massachusetts. 2000. 481p. </li>
<li>Hedges, S. B.; Kumar, S. (Eds). The timetree of life. Oxford University Press, New York. 2009.</li>
<li>Higgins, D., Lemey, P.. Multiple sequence alignment. In Lemey, P., Salemi, M., Vandamme, A. (eds.). The Phylogenetic Handbook: a Practical Approach to Phylogenetic Analysis and Hypothesis Testing, Published by Cambridge University Press. Cambridge University Press 2009. </li>
<li>Ho, S. Y. W.; Shapiro, B. Skyline-plot methods for estimating demographic history from nucleotide sequences. Molecular Ecology Resources, v. 11, p. 423–434, 2011. </li>
<li>Judd, W. S.; Campbell, C. S. &amp; Stevens, P. F. Plant Systematics: A Phylogenetic Approach. Sinauer Associates, Inc. Publishers. Sunderland, MA, USA. 1999. 576p. </li>
<li>Kumar S, Stecher G, Tamura K. MEGA7: Molecular Evolutionary Genetics Analysis Version 7.0 for Bigger Datasets. Molecular Biology and Evolution. 2016;33(7):1870-1874. doi:10.1093/molbev/msw054.</li>
<li>Kuraku S, Feiner N, Keeley SD, Hara Y. Incorporating tree-thinking and evolutionary time scale into developmental biology. HANASHIMA C, NISHIMURA T, NAKAMURA H, STERN C, eds. Develop Growth Differ. 2016;58(1):131-142. doi:10.1111/dgd.12258.</li>
<li>Lemey, P., Posada, D. S. Molecular clock analysis. In Lemey, P., Salemi, M., Vandamme, A. (eds.). The Phylogenetic Handbook: a Practical Approach to Phylogenetic Analysis and Hypothesis Testing, Published by Cambridge University Press. Cambridge University Press 2009.</li>
<li>Luís Q Alves, Raquel Ruivo, Miguel M Fonseca, Mónica Lopes-Marques, Pedro Ribeiro, L Filipe C Castro, Pseudo<em>Checker</em>: an integrated online platform for gene inactivation inference, <em>Nucleic Acids Research</em>, Volume 48, Issue W1, 02 July 2020, Pages W321–W331, https://doi.org/10.1093/nar/gkaa408</li>
<li>Mayr, E. O Desenvolvimento do Pensamento Biológico: Diversidade, Evolução e Herança. Editora UnB. Universidade de Brasília. 1998. 1007p. </li>
<li>Morrison DA. L. A. S. JOHNSON REVIEW No. 8. Multiple sequence alignment for phylogenetic purposes. Aust Systematic Bot. 2006;19(6):479. doi:10.1071/SB06020.</li>
<li>Nei, M. &amp; Kumar, S. Molecular Evolution and Phylogenetics. Oxford University Press, Inc. 2001. 333p. </li>
<li>Omland KE, Cook LG, Crisp MD. Tree thinking for all biology: the problem with reading phylogenies as ladders of progress. Bioessays. 2008;30(9):854-867. doi:10.1002/bies.20794.</li>
<li>Page, R. D. M. &amp; Holmes, E. C. Molecular Evolution: A phylogenetic approach. Blackwell Science. 2001. 346 pp. </li>
<li>Kolekar, P., Kale, M., Kulkarni-Kale, U. (2011). Molecular Evolution &amp; Phylogeny: What, When, Why &amp; How?, Computational Biology and Applied Bioinformatics, Prof. Heitor Lopes (Ed.), InTech, DOI: 10.5772/20225.</li>
<li>Pond, S. L. K., Poon, A. F. Y., Frost, S. D. W. Estimating selection pressures on alignments of coding sequences. In Lemey, P., Salemi, M., Vandamme, A. (eds.). The Phylogenetic Handbook: a Practical Approach to Phylogenetic Analysis and Hypothesis Testing, Published by Cambridge University Press. Cambridge University Press 2009.</li>
<li>Posada, D. &amp; Crandall, K. A. ModelTest: Testing the model of DNA substitution. Bioinformatics v. 14, n. 9, p. 817-818. 1998. </li>
<li>Posada, D. Selecting models of evolution. In Lemey, P., Salemi, M., Vandamme, A. (eds.). The Phylogenetic Handbook: a Practical Approach to Phylogenetic Analysis and Hypothesis Testing, Published by Cambridge University Press. Cambridge University Press 2009.</li>
<li>Rieppel, O.: The series, the network, and the tree: changing metaphors of order in nature. - Biol. Phil. 25: 475-496, 2010.</li>
<li>Russo, C. A. de M., Miyaki, C. Y. &amp; Pereira, S. L. Reconstrução Filogenética. Métodos Geométricos. In: Matioli, S. R. (ed.) Biologia Molecular e Evolução. p. 108 -116. Holos editora. Ribeirão Preto, SP. 2001.</li>
<li>Sakamoto, T., Ortega, J.M. TaxOnTree: a tool that generates trees annotated with taxonomic information. bioRxiv 2020. DOI: <a href="https://doi.org/10.1101/2020.12.24.424364">10.1101/2020.12.24.424364</a>.</li>
<li>Sttebins, G. L. Processos de Evolução Orgânica. Editora Polígono e Editora da Universidade de São Paulo. São Paulo. 1970. 253p. </li>
<li>Van de Peer, Y.  Phylogenetic inference based on distance methods. In - Lemey, P., Salemi, M., Vandamme, A. (eds.). The Phylogenetic Handbook: a Practical Approach to Phylogenetic Analysis and Hypothesis Testing, Published by Cambridge University Press. Cambridge University Press 2009.</li>
<li>Xia X. DAMBE6: New Tools for Microbial Genomics, Phylogenetics, and Molecular Evolution. Journal of Heredity. 2017;108(4):431-437. doi:10.1093/jhered/esx033.</li>
<li>Xia, X. 2000. Data Analysis in Molecular Biology and Evolution. Kluwer Academic Publishers.</li>
<li>Xia, X. Assessing substitution saturation with Dambe. In Lemey, P., Salemi, M., Vandamme, A. (eds.). The Phylogenetic Handbook: a Practical Approach to Phylogenetic Analysis and Hypothesis Testing, Published by Cambridge University Press. Cambridge University Press 2009.</li>
<li>Yang Z, Rannala B. Molecular phylogenetics: principles and practice. Nature Review Genetics. 2012;13(303):1-12. doi:10.1038/nrg3186.</li>
</ul>
              
            </div>
          </div><footer>
    <div class="rst-footer-buttons" role="navigation" aria-label="Footer Navigation">
        <a href="../UC_slides/" class="btn btn-neutral float-left" title="Slides"><span class="icon icon-circle-arrow-left"></span> Previous</a>
        <a href="../First_Tutorial_ENG/" class="btn btn-neutral float-right" title="First Tutorial">Next <span class="icon icon-circle-arrow-right"></span></a>
    </div>

  <hr/>

  <div role="contentinfo">
    <!-- Copyright etc -->
  </div>

  Built with <a href="https://www.mkdocs.org/">MkDocs</a> using a <a href="https://github.com/readthedocs/sphinx_rtd_theme">theme</a> provided by <a href="https://readthedocs.org">Read the Docs</a>.
</footer>
          
        </div>
      </div>

    </section>

  </div>

  <div class="rst-versions" role="note" aria-label="Versions">
  <span class="rst-current-version" data-toggle="rst-current-version">
    
    
      <span><a href="../UC_slides/" style="color: #fcfcfc">&laquo; Previous</a></span>
    
    
      <span><a href="../First_Tutorial_ENG/" style="color: #fcfcfc">Next &raquo;</a></span>
    
  </span>
</div>
    <script src="../js/jquery-3.6.0.min.js"></script>
    <script>var base_url = "..";</script>
    <script src="../js/theme_extra.js"></script>
    <script src="../js/theme.js"></script>
      <script src="../search/main.js"></script>
    <script>
        jQuery(function () {
            SphinxRtdTheme.Navigation.enable(true);
        });
    </script>

</body>
</html>
