<!DOCTYPE html>
<html class="writer-html5" lang="en" >
<head>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" /><meta name="author" content="jpmslima" /><link rel="canonical" href="https://jpmslima.github.io/Similarity%20Search/" />
      <link rel="shortcut icon" href="../img/favicon.ico" />
    <title>Similarity searches using protein sequences - Tutoriais BioinfoX</title>
    <link rel="stylesheet" href="../css/theme.css" />
    <link rel="stylesheet" href="../css/theme_extra.css" />
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.8.0/styles/github.min.css" />
    
      <script>
        // Current page data
        var mkdocs_page_name = "Similarity searches using protein sequences";
        var mkdocs_page_input_path = "Similarity Search.md";
        var mkdocs_page_url = "/Similarity%20Search/";
      </script>
    
    <!--[if lt IE 9]>
      <script src="../js/html5shiv.min.js"></script>
    <![endif]-->
      <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.8.0/highlight.min.js"></script>
        <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.8.0/languages/yaml.min.js"></script>
      <script>hljs.highlightAll();</script> 
</head>

<body class="wy-body-for-nav" role="document">

  <div class="wy-grid-for-nav">
    <nav data-toggle="wy-nav-shift" class="wy-nav-side stickynav">
    <div class="wy-side-scroll">
      <div class="wy-side-nav-search">
          <a href=".." class="icon icon-home"> Tutoriais BioinfoX
        </a><div role="search">
  <form id ="rtd-search-form" class="wy-form" action="../search.html" method="get">
      <input type="text" name="q" placeholder="Search docs" aria-label="Search docs" title="Type search term here" />
  </form>
</div>
      </div>

      <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="Navigation menu">
              <ul>
                <li class="toctree-l1"><a class="reference internal" href="..">Página Inicial</a>
                </li>
              </ul>
              <ul>
                <li class="toctree-l1"><a class="reference internal" href="../demo/">Representação Computacional de Biomoléculas</a>
                </li>
              </ul>
              <ul>
                <li class="toctree-l1"><a class="reference internal" href="../aboutus/">Sobre o grupo</a>
                </li>
              </ul>
              <p class="caption"><span class="caption-text">Tutoriais Iniciais de Bioinformática</span></p>
              <ul>
                  <li class="toctree-l1"><a class="reference internal" href="../basic1/">Inicial</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../101/">Introdução a linha de comando UNIX</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../102/">Manipulando dados na CLI</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../CLI-Colab/">Introdução a CLI e ao Google Colab</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../103/">Introdução ao GenBank</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../104/">Buscas de Similaridade</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../105/">Alinhamento entre pares de sequências</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../106-MSA/">Alinhamento Múltiplo de Sequências</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../107-NCBI-CLI/">NCBI/GenBank via CLI</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../108-BLAST-CLI/">BLAST via CLI</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../109-SQL/">Comandos Básicos em SQL</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../r-colab/">Introdução ao R usando o Google Colab</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../enzima-annot/">Integração de Bancos - Exemplo</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../dbsnp/">Bancos de dados de Mutações e Polimorfismos</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../KEGG/">Explorando Vias Metabólicas</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../110-BDs/">Bancos de Dados de Informação Proteica</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../111-Modelagem/">Modelagem Computacional de Proteínas</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../112-MEGAX/">Primeiros Passos no MEGA</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../113-Genomes/">Navegando por Genomas</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../video/">Videoaulas</a>
                  </li>
              </ul>
              <p class="caption"><span class="caption-text">Bioinformática Estrutural</span></p>
              <ul>
                  <li class="toctree-l1"><a class="reference internal" href="../basic2/">Inicial</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../slides/">Slides de Acompanhamento</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../tools/">Programas e Ferramentas</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../STR1-Visualizacao/">Visualização Computacional de Moléculas</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../STR2-pred/">Informações a partir da estrutura primária</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../STR22-Epitopos/">Predição de Epítopos</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../STR3-Model1/">Modelagem 101</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../STR4-Model2/">Modelagem 102</a>
                  </li>
                  <li class="toctree-l1"><a class="" href="../STR5-Model3.mkdocs gh-deploy --forcemd">Modeller-CLI</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../STR6-AlphaFold/">Métodos baseados em IA</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../STR7-MDA/">MDA</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../STR8-Aval/">Avaliação dos Modelos</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../STR9-EM/">EM/CryoEM fitting</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../STR10-RINs/">RINs</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../STR11-APBS/">APBS</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../STR12-Memb/">Proteínas de Membrana</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../STR13-Dock1/">Autodock (básico)</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../STR14-Dock2/">Docking</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../STR15-VS1/">Virtual Screening</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../STR23-MDIntro/">Introdução a Dinâmica Molecular</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../STR16-EGB1/">Modelos de Pequenas Moléculas</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../STR17-EGB2/">Análise conformacional</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../STR18-EGB3/">Docking Automático</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../STR19-EGB4/">Posicionamento Manual</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../STR20-EGB5/">Dinâmica Molecular</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../STR21-EGB6/">VS-2</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../comps/">Complements</a>
                  </li>
              </ul>
              <p class="caption"><span class="caption-text">Análise Evolucionária de Sequências (Evolução Molecular)</span></p>
              <ul>
                  <li class="toctree-l1"><a class="reference internal" href="../satsub-ptbr/">Análise de Saturação de Substituições</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../bestfit-ptbr/">Escolha do Modelo de Substituição</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../BEAST_ptbr/">Usando o BEAST</a>
                  </li>
              </ul>
              <p class="caption"><span class="caption-text">Diplomado - UChile</span></p>
              <ul>
                  <li class="toctree-l1"><a class="reference internal" href="../Presentation/">Home</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../UC_slides/">Slides</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../Material/">Files and References</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../First_Tutorial_ENG/">First Tutorial</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../Substitution_saturation_ENG/">Analysis of Substitution Saturation</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../ModelTest_ENG/">Test of the Best-Fit Models</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../MEGAX_ENG/">Using MEGAX</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../1st_BEAST/">Using BEAST</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../BEAST%2BEBSP_ENG/">Phylodynamics using EBSP</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../IQTree/">Using IQ-Tree</a>
                  </li>
              </ul>
      </div>
    </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">
      <nav class="wy-nav-top" role="navigation" aria-label="Mobile navigation menu">
          <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
          <a href="..">Tutoriais BioinfoX</a>
        
      </nav>
      <div class="wy-nav-content">
        <div class="rst-content"><div role="navigation" aria-label="breadcrumbs navigation">
  <ul class="wy-breadcrumbs">
    <li><a href=".." class="icon icon-home" aria-label="Docs"></a></li>
      <li class="breadcrumb-item active">Similarity searches using protein sequences</li>
    <li class="wy-breadcrumbs-aside">
    </li>
  </ul>
  <hr/>
</div>
          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
            <div class="section" itemprop="articleBody">
              
                <h1 id="similarity-searches-using-protein-sequences">Similarity searches using protein sequences</h1>
<p>In the following steps, you should learn:</p>
<ul>
<li>Searching for similarities among sequences using the different BLAST versions.</li>
<li>Discover information about a protein using the primary structure (sequence).</li>
<li>Search for homologous proteins that have low sequence similarity.</li>
</ul>
<p><strong>Notice:</strong>
- This tutorial was adapted from the BLAST tutorials available at NCBI. The original content has been translated and modified in part its structure <strong><em>only for didactic purposes</em></strong>. <span style="color:red"><strong>Any reproduction of it for any other purpose is neither permitted nor consented to by the course instructor.</strong></span></p>
<h2 id="blastp">BLASTp</h2>
<p>Use <a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?PROGRAM=blastp&amp;PAGE_TYPE=BlastSearch&amp;LINK_LOC=blasthome">BLASTp</a>, to run a search against the <strong>nr</strong> bank (<em>non redundant protein database</em>) to find vertebrate proteins homologous to the protein below:</p>
<pre><code>&gt;C.elegans protein
MFHPGMTSQPSTSNQMYYDPLYGAEQIVQCNPMDYHQANILCGMQYFNNSHNRYPLLPQMPPQFTNDHPY
DFPNVPTISTLDEASSFNGFLIPSQPSSYNNNNISCVFTPTPCTSSQASSQPPPTPTVNPTPIPPNAGAV
LTTAMDSCQQISHVLQCYQQGGEDSDFVRKAIESLVKKLKDKRIELDALITAVTSNGKQPTGCVTIQRSL
DGRLQVAGRKGVPHVVYARIWRWPKVVSKNELVKLVQCQTSSDHPDNICINPYHYERVVSNRITSADQSLH
VENSPMKSEYLGDAGVIDSCSDWPNTPPDNNFNGGFAPDQPQLVTPIISDIPIDLNQIYVPTPPQLLDNW
CSIIYYELDTPIGETFKVSARDHGKVIVDGGMDPHGENEGRLCLGALSNVHRTEASEKARIHIGRGVELT
AHADGNISITSNCKIFVRSGYLDYTHGSEYSSKAHRFTPNESSFTVFDIRWAYMQMLRRSRSSNEAVRAQ
AAAVAGYAPMSVMPAIMPDSGVDRMRRDFCTIAISFVKAWGDVYQRKTIKETPCWIEVTLHRPLQILDQL
LKNSSQFGSS
</code></pre>
<p>After a few minutes, you should have a result like the one below. Click on the place indicated to switch to the traditional BLAST results page:</p>
<p><img alt="Example of a BLASTn result between 2 sequences" src="https://drive.google.com/uc?id=14tdccH2BgW8ksapmRVlkH6A468psJq9a" /></p>
<blockquote>
<p><em>On the traditional page, the results are more accessible, however, feel free to use the newer version.</em></p>
</blockquote>
<ul>
<li>Based on the <em>hit</em> identical to <em>C. elegans</em>, what is the identity of this protein?</li>
</ul>
<p>Above the graphical summary of the BLAST search is the link <strong><em>Taxonomy Report</em></strong> (See figure below). Click and look at the new page that opens.</p>
<p><img alt="Taxonomy Report" src="https://drive.google.com/uc?id=1I58CAsoPWIrU7h86d2lEEIpLaBdhhzfU" /></p>
<ul>
<li>What do you notice on this new page?</li>
<li>Look on this page for records of homologous proteins of bird species.</li>
<li>Look for the hit with the species <em>Xenopus laevis</em> and check the BLAST alignment parameters between these sequences.</li>
</ul>
<blockquote>
<p><em>We will come back to this protein in the PSI-BLAST example.</em></p>
</blockquote>
<h2 id="blastx">BLASTx</h2>
<p>The easiest way to obtain a target sequence is to search its coding sequence in biological databases (in genome or transcriptome projects). DNA sequencing is usually more accessible than methods of obtaining amino acid sequences. To perform the similarity search from the nucleotide coding sequence, you have two options:</p>
<ul>
<li>Translate the nucleotide sequence in the 3 <em>frames</em> (open read frames) and submit the resulting sequences to similarity searches using BLASTp. This translation can be performed using tools from <a href="https://web.expasy.org/cgi-bin/translate/dna2aa.cgi">EXPASY</a> or <a href="https://www.ebi.ac.uk/Tools/st/">EBI</a>.</li>
<li>Use <a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?PROGRAM=blastx&amp;PAGE_TYPE=BlastSearch&amp;BLAST_SPEC=&amp;LINK_LOC=blasttab&amp;LAST_PAGE=blastn&amp;QUERY=AB823629.1">BLASTx</a>. This variation of BLAST translates the input nucleotide sequence (<em>Query</em>) and compares it with protein banks.</li>
</ul>
<p>To be more practical and straightforward, we will use the search <a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?PROGRAM=blastx&amp;PAGE_TYPE=BlastSearch&amp;BLAST_SPEC=&amp;LINK_LOC=blasttab&amp;LAST_PAGE=blastn&amp;QUERY=AB823629.1">BLASTx</a>, with the sequence in the <code>fasta</code> format below:</p>
<pre><code>&gt;AB823629.1
GAATGCATACATGGCTTCCTCCAACTTACTCTCCCTAGCCCTCTTCCTTGTGCTTCTCACCCACGCAAAC
TCAGCCAGCCAAACCTCCTTCAGCTTCCAAAGGTTCAACGAAACCAACCTTATCCTCCAACGCGATGCCA
CCGTCTCATCCAAAGGCCAGTTACGACTAACCAATGTTAATGACAACGGAGAACCCACGTTGAGCTCTCT
GGGCCGTGCCTTCTACTCCGCCCCCATCCAAATCTGGGACAACACCACCGGCGCCGTGGCCAGCTTCGCC
ACCTCCTTCACATTCAATATCGACGTTCCCAACAATTCAGGACCCGCCGATGGCCTTGCCTTTGTTCTCC
TCCCCGTGGGCTCTCAGCCCAAAGACAAAGGCGGTCTTCTAGGTCTGTTCAACAACTACAAATACGACAG
CAATGCCCATACTGTGGCTGTGGAGTTCGACACCCTCTACAACGTTCACTGGGACCCCAAACCGCGTCAT
ATTGGCATCGACGTGAACTCCATCAAGTCTATCAAAACGACGACGTGGGATTTTGTCAAAGGAGAAAACG
CGGAGGTTCTGATCACCTATGACTCCTCCACGAAGCTCTTGGTGGCTTCTCTGGTTTACCCTTCTCTGAA
AACAAGCTTCATCGTCTCTGACACAGTGGACCTGAAGAGCGTTCTTCCCGAGTGGGTGATCGTTGGGTTC
ACTGCCACCACTGGGATTACTAAAGGGAACGTTGAAACGAACGACATCCTCTCTTGGTCTTTTGCTTCCA
AGCTCTCCGATGGCACCACATCTGAAGCTTTGAATCTTGCCAACTTCGCCCTCAACCAAATCCTCTAG
</code></pre>
<p>Now let's see if there is other proteins similar to this one that have an experimentally known 3D structure. For this we will use a <a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?PROGRAM=blastx&amp;PAGE_TYPE=BlastSearch&amp;BLAST_SPEC=&amp;LINK_LOC=blasttab&amp;LAST_PAGE=blastn&amp;QUERY=AB823629.1">BLASTx</a>. On the BLASTx search page, remember to change the <em>Database</em> field to <strong><em>Protein Data Bank (pdb)</em></strong>. However, we do not want to receive <em>hits</em> from the genus <em>Phaseolus</em>. To do this, in the <em>Organism</em> field, we put the name <em>Phaseolus</em> (taxid: 3883) and click <em>Exclude</em> (See the figure below).</p>
<p><img alt="BlastX" src="https://drive.google.com/uc?id=1SdKtOtwJ4NP7V1yNk36JXmymZJqhBh0R" /></p>
<p>Click BLAST and wait for the results.
- What is the percent identity of the pdb entry most similar to this sequence?</p>
<h2 id="psi-blast">PSI-BLAST</h2>
<p>The Sma-4 protein from <em>C. elegans</em> that we used earlier belongs to a large family of proteins involved in TGF-beta mediated signaling. (Go back to the BLASTp search from the beginning of this tutorial). Some members of this family are not readily identified in a normal BLASTp search, because they already had many modifications over evolutionary time in their sequences. However, other more distant homologs of Sma-4 can be found using the more specific PSI-BLAST sensitive to the specific amino acid position in the proteins. </p>
<p>Start a PSI-BLAST search with the accession number for the <em>C. elegans</em> Sma-4 protein, <strong>P45897.1</strong>, as a query. To use PSI-BLAST, you start as an ordinary BLASTp and select the PSI-BLAST option below (see the figure below):</p>
<p><img alt="PSI-BLAST" src="https://drive.google.com/uc?id=14QUOcRl8ZjvOhX7g1Pn0m9erL3NAlCek" /></p>
<p>Notice the first output of results. It does not differ much from the output of a normal BLASTp, except that it is formatted differently. The sequences are already marked/selected. There is a line between results corresponding to the PSI-BLAST inclusion threshold of 0.005. The position-specific information from a multiple alignment of the sequences above this line is used to generate a position-specific score matrix (PSSM) in the next iteration.</p>
<p>Now we will do a new round of iteration. In this one the program will calculate a new PSSM and will look for proteins that have less similar sequences, but still respect the probabilities from the PSSM. Now click the <em>Go</em> button next to <em>Run PSI-BLAST iteration 2</em>. </p>
<p>The hits not found in the previous search will appear highlighted in yellow.</p>
<ul>
<li>What is the first yellow <em>hit</em> found? What is the sequence similarity?</li>
</ul>
<p>You can continue by performing new iteration steps. After a few more iterations no more new sequences will be found. At this point it is said that the search has reached convergence.</p>
<blockquote>
<p><em>In recent years the NCBI has launched the Delta-BLAST tool, which gives similar results to PSI-BLAST, but in a faster way and more integrated with the regular BLAST.</em></p>
</blockquote>
<h2 id="phi-blast">PHI-BLAST</h2>
<p>PHI-BLAST searches for sequences that have a certain pattern or signature. Let's initially do a BLASTp search, using the sequence below:</p>
<pre><code>&gt;Protein_X
MCPANIAPVCCYGYSTGSYTTPNVCLCKCVLDTFVLYEGLCRITHDKFKPPRDDILCLQVYDPVCCRIHGVVVTESNGCFCYVRGGTVVPGTCSSMFPPPFASPQIS
</code></pre>
<ul>
<li>How many <em>hits</em> were found?</li>
</ul>
<p>Although only these sequences were found, there may be other proteins that have a similar sequence signature to these proteins. From a multiple alignment of the obtained proteins, the following sequence signature was assembled:</p>
<pre><code>C-x(5)-PVCC-x(1,4)-G-x(1,6)-T-x(2)-N-x(1)-C-x(7,14)-G-x(1)-C-x(1,5)-[HN]-x(4)-P
</code></pre>
<blockquote>
<p><em>We will not show this procedure now. What can be described is that they correspond to regions that have the same amino acids, separated by very different ones. This signature has to be written in the format of the <a href="http://prosite.expasy.org">PROSITE</a> database.</em></p>
</blockquote>
<p>Now we will use this pattern, as a search pattern in PHI-BLAST. On the BLASTp home page, select <strong>PHI-BLAST</strong> (it is just below PSI-BLAST - see the picture below). Clicking on PSI-BLAST opens a box below. In it you enter the signature above. Click BLAST and wait for the results (It can take a while).</p>
<p><img alt="PHI-BLAST" src="https://drive.google.com/uc?id=1V3svK_WdQk4YtbF7rykFK54O3Es5PJC-" /></p>
<ul>
<li>Did new proteins appear in this search?</li>
<li>How similar are these new proteins?</li>
</ul>
              
            </div>
          </div><footer>

  <hr/>

  <div role="contentinfo">
    <!-- Copyright etc -->
  </div>

  Built with <a href="https://www.mkdocs.org/">MkDocs</a> using a <a href="https://github.com/readthedocs/sphinx_rtd_theme">theme</a> provided by <a href="https://readthedocs.org">Read the Docs</a>.
</footer>
          
        </div>
      </div>

    </section>

  </div>

  <div class="rst-versions" role="note" aria-label="Versions">
  <span class="rst-current-version" data-toggle="rst-current-version">
    
    
    
  </span>
</div>
    <script src="../js/jquery-3.6.0.min.js"></script>
    <script>var base_url = "..";</script>
    <script src="../js/theme_extra.js"></script>
    <script src="../js/theme.js"></script>
      <script src="../search/main.js"></script>
    <script>
        jQuery(function () {
            SphinxRtdTheme.Navigation.enable(true);
        });
    </script>

</body>
</html>
