![UChile](UChile.png)

# Files

## Working *Datasets*

**Tree Files for use in FigTree:**

- [Tree File](https://drive.google.com/uc?export=download&id=1CPz5isUp25TEY9ZHf9R4adBNu-IBn51c)
- [Tree File 2](https://drive.google.com/uc?export=download&id=1FjK3rTahhtx7paGs0zemJh4OyNZgAk4B)

**Datasets for multiple sequence alignment:**

- [primatesNuc_1.fasta](https://drive.google.com/uc?export=download&id=1nUkT_zqkvHoqR5x7N5ggLmnMNb_982GH)
- [primatesAA_1.fasta](https://drive.google.com/uc?export=download&id=1qJMyqjvd6SIz6SAeeggHZdPIb9do8CIg)

**For use in the analysis of substitution saturation:**

- [VertCOI.fas](https://drive.google.com/uc?export=download&id=1sPodmXGkPCxWu7t8CIZzHmumYS_0kX18)
- [InvertebrateEF1a.fas](https://drive.google.com/uc?export=download&id=1pDxXpGbrWawiwBnhf3sZxi4EFGA-4Di-)
- [SIV.fas](https://drive.google.com/uc?export=download&id=1d99-xXZDWTE6nhAuI439T5VzHhTbeL2P)

**Selection tests**

- [Data I.zip](https://drive.google.com/uc?export=download&id=1GncrP4SzFtV79LogQ-btK18EIltQHRTL)

> *Additional tutorial.*

**For use in the BEAST+EBSP tutorial:**

- [Tutorial_Zika_Filogenia.zip](https://drive.google.com/uc?export=download&id=19xrCmUwQ94JglHMU5BMjoNEAXRuK_jl6)

## Programs, Applications and Tools

- [MEGAX](http://www.megasoftware.net)
- [DAMBE](http://dambe.bio.uottawa.ca/DAMBE/dambe.aspx)
- [Bioedit](http://www.mbio.ncsu.edu/BioEdit/bioedit.html)
- [gedit](https://wiki.gnome.org/Apps/Gedit)
- [jModelTest](https://github.com/ddarriba/jmodeltest2)
- [Prottest 3](https://github.com/ddarriba/prottest3)
- [ModelTest-ng](https://github.com/ddarriba/modeltest)
- [BEAST2](https://www.beast2.org)
- [BEAST 1.8](http://beast.bio.ed.ac.uk/downloads)
- [Clustal X](http://clustal.org/download/current/)
- [PAMLX](http://abacus.gene.ucl.ac.uk/software/paml.html#download)
- [R](https://www.r-project.org)
- [Aliview](http://www.ormbunkar.se/aliview/)
- [FigTree](http://tree.bio.ed.ac.uk/software/figtree/)
- [Jalview](http://www.jalview.org)
- [TrimAL](http://trimal.cgenomics.org)

## *Online* Tools

- [TaxOnTree](http://bioinfo.icb.ufmg.br/taxontree/#x)

> *Comparative and exploratory phylogeny, as well as phylogenetic trees annottion.*

- [T-Coffee](http://tcoffee.crg.cat)

> *Alignment program. Several options depending on your dataset.*

- [Phylemon2](http://phylemon2.bioinfo.cipf.es)

> *Site with on line phylogenetics tools and pipelines. There, you can use TrimAL to "clean" gappy alignments.*

- [DataMonkey](https://www.datamonkey.org)

> *Several tools used in selection tests, recombination and gene conversion detection.*

- [Pseudochecker](http://pseudochecker.ciimar.up.pt/pseudochecker/index.html)

> *A tool to infer eukaryotic nuclear protein-coding genes inactivation.*

- [CIPRES](http://www.phylo.org/)

> *Public portal containing pipelines for inferring phylogenies.*

## Additional *Links*

- [TimeTree](http://www.timetree.org)
- [Phylogeny.fr](http://www.phylogeny.fr/index.cgi)
- [Nextstrain](https://nextstrain.org)
- [GSAID](https://www.gisaid.org)
- [Virus Variation](https://www.ncbi.nlm.nih.gov/genome/viruses/variation/)
- [BEAST 1.X Tutorials](https://beast.community)
- [BEAST 2.X Tutorials](https://www.beast2.org/tutorials/)

## References

- Anisimova M, Liberles DA, Philippe H, Provan J, Pupko T, Haeseler von A. State-of the art methodologies dictate new standards for phylogenetic analysis. BMC Evol Biol. 2013;13(1):1-1. doi:10.1186/1471-2148-13-161.
- Higgs, P.G.; Attwood, T.K. Bioinformatics and Molecular Evolution.. Malden (Massachusetts): Blackwell Publishing. 2005.
- Bouckaert, R. et al. BEAST 2: A Software Platform for Bayesian Evolutionary Analysis. PLoS Computational Biology, v. 10, n. 4, p. e1003537, 10 abr. 2014.
- Choudhuri S. Bioinformatics for Beginners: Genes, Genomes, Molecular Evolution, Databases and Analytical Tools. Elsevier, 2014.
- Cvrckova F. A plant biologists’ guide to phylogenetic analysis of biological macromolecule sequences. Biologia Plantarum. 2016;60(4):619-627.
- Dickinson H, Costa L, Gutierrez-Marcos J. Epigenetic neofunctionalisation and regulatory gene evolution in grasses. Trends in Plant Science. 2012;17(7):389-394.
- Do CB, Katoh K. Protein Multiple Sequence Alignment. Thompson JD, Schaeffer-Reiss C, Ueffing M, eds. 2008:379-413. doi:10.1007/978-1-59745-398-1_25.
- Drummond, A. J. et al. Bayesian coalescent inference of past population dynamics from molecular sequences. Molecular Biology and Evolution, v. 22, p. 1185–1192, 2005. 
- Edgar RC, Batzoglou S. Multiple sequence alignment. Current Opinion in Structural Biology. 2006;16(3):368-373. doi:10.1016/j.sbi.2006.04.004.
- Faria, N. R. et al. Zika virus in the Americas: Early epidemiological and genetic findings. Science (New York, N.Y.), v. 5036, n. March, p. aaf5036, 2016. 
- Futuyma DJ. Evolution. Sunderland, Massachusetts, USA.: Sinauer Associates; 2nd Edition. 2009.
- Gardner PP. A benchmark of multiple sequence alignment programs upon structural RNAs. Nucleic Acids Research. 2005;33(8):2433-2439. doi:10.1093/nar/gki541.
- Gibas, c. & jambeck, P. Developing Bioinformatics Computer Skills. O ́Reilly & Associates, Inc. USA. 2001. 
- Graur, d. & Li, W. H. Fundamentals of Molecular Evolution. 2a edição. Sinauer Press, Sunderland, Massachusetts. 2000. 481p. 
- Hedges, S. B.; Kumar, S. (Eds). The timetree of life. Oxford University Press, New York. 2009.
- Higgins, D., Lemey, P.. Multiple sequence alignment. In Lemey, P., Salemi, M., Vandamme, A. (eds.). The Phylogenetic Handbook: a Practical Approach to Phylogenetic Analysis and Hypothesis Testing, Published by Cambridge University Press. Cambridge University Press 2009. 
- Ho, S. Y. W.; Shapiro, B. Skyline-plot methods for estimating demographic history from nucleotide sequences. Molecular Ecology Resources, v. 11, p. 423–434, 2011. 
- Judd, W. S.; Campbell, C. S. & Stevens, P. F. Plant Systematics: A Phylogenetic Approach. Sinauer Associates, Inc. Publishers. Sunderland, MA, USA. 1999. 576p. 
- Kumar S, Stecher G, Tamura K. MEGA7: Molecular Evolutionary Genetics Analysis Version 7.0 for Bigger Datasets. Molecular Biology and Evolution. 2016;33(7):1870-1874. doi:10.1093/molbev/msw054.
- Kuraku S, Feiner N, Keeley SD, Hara Y. Incorporating tree-thinking and evolutionary time scale into developmental biology. HANASHIMA C, NISHIMURA T, NAKAMURA H, STERN C, eds. Develop Growth Differ. 2016;58(1):131-142. doi:10.1111/dgd.12258.
- Lemey, P., Posada, D. S. Molecular clock analysis. In Lemey, P., Salemi, M., Vandamme, A. (eds.). The Phylogenetic Handbook: a Practical Approach to Phylogenetic Analysis and Hypothesis Testing, Published by Cambridge University Press. Cambridge University Press 2009.
- Luís Q Alves, Raquel Ruivo, Miguel M Fonseca, Mónica Lopes-Marques, Pedro Ribeiro, L Filipe C Castro, Pseudo*Checker*: an integrated online platform for gene inactivation inference, *Nucleic Acids Research*, Volume 48, Issue W1, 02 July 2020, Pages W321–W331, https://doi.org/10.1093/nar/gkaa408
- Mayr, E. O Desenvolvimento do Pensamento Biológico: Diversidade, Evolução e Herança. Editora UnB. Universidade de Brasília. 1998. 1007p. 
- Morrison DA. L. A. S. JOHNSON REVIEW No. 8. Multiple sequence alignment for phylogenetic purposes. Aust Systematic Bot. 2006;19(6):479. doi:10.1071/SB06020.
- Nei, M. & Kumar, S. Molecular Evolution and Phylogenetics. Oxford University Press, Inc. 2001. 333p. 
- Omland KE, Cook LG, Crisp MD. Tree thinking for all biology: the problem with reading phylogenies as ladders of progress. Bioessays. 2008;30(9):854-867. doi:10.1002/bies.20794.
- Page, R. D. M. & Holmes, E. C. Molecular Evolution: A phylogenetic approach. Blackwell Science. 2001. 346 pp. 
- Kolekar, P., Kale, M., Kulkarni-Kale, U. (2011). Molecular Evolution & Phylogeny: What, When, Why & How?, Computational Biology and Applied Bioinformatics, Prof. Heitor Lopes (Ed.), InTech, DOI: 10.5772/20225.
- Pond, S. L. K., Poon, A. F. Y., Frost, S. D. W. Estimating selection pressures on alignments of coding sequences. In Lemey, P., Salemi, M., Vandamme, A. (eds.). The Phylogenetic Handbook: a Practical Approach to Phylogenetic Analysis and Hypothesis Testing, Published by Cambridge University Press. Cambridge University Press 2009.
- Posada, D. & Crandall, K. A. ModelTest: Testing the model of DNA substitution. Bioinformatics v. 14, n. 9, p. 817-818. 1998. 
- Posada, D. Selecting models of evolution. In Lemey, P., Salemi, M., Vandamme, A. (eds.). The Phylogenetic Handbook: a Practical Approach to Phylogenetic Analysis and Hypothesis Testing, Published by Cambridge University Press. Cambridge University Press 2009.
- Rieppel, O.: The series, the network, and the tree: changing metaphors of order in nature. - Biol. Phil. 25: 475-496, 2010.
- Russo, C. A. de M., Miyaki, C. Y. & Pereira, S. L. Reconstrução Filogenética. Métodos Geométricos. In: Matioli, S. R. (ed.) Biologia Molecular e Evolução. p. 108 -116. Holos editora. Ribeirão Preto, SP. 2001.
- Sakamoto, T., Ortega, J.M. TaxOnTree: a tool that generates trees annotated with taxonomic information. bioRxiv 2020. DOI: [10.1101/2020.12.24.424364](https://doi.org/10.1101/2020.12.24.424364).
- Sttebins, G. L. Processos de Evolução Orgânica. Editora Polígono e Editora da Universidade de São Paulo. São Paulo. 1970. 253p. 
- Van de Peer, Y.  Phylogenetic inference based on distance methods. In - Lemey, P., Salemi, M., Vandamme, A. (eds.). The Phylogenetic Handbook: a Practical Approach to Phylogenetic Analysis and Hypothesis Testing, Published by Cambridge University Press. Cambridge University Press 2009.
- Xia X. DAMBE6: New Tools for Microbial Genomics, Phylogenetics, and Molecular Evolution. Journal of Heredity. 2017;108(4):431-437. doi:10.1093/jhered/esx033.
- Xia, X. 2000. Data Analysis in Molecular Biology and Evolution. Kluwer Academic Publishers.
- Xia, X. Assessing substitution saturation with Dambe. In Lemey, P., Salemi, M., Vandamme, A. (eds.). The Phylogenetic Handbook: a Practical Approach to Phylogenetic Analysis and Hypothesis Testing, Published by Cambridge University Press. Cambridge University Press 2009.
- Yang Z, Rannala B. Molecular phylogenetics: principles and practice. Nature Review Genetics. 2012;13(303):1-12. doi:10.1038/nrg3186.
